pages/linux/stress.md

```diff
@@ -0,0 +1,19 @@
+# stress
+
+> A tool to stress test CPU, memory, and IO on a Linux system.
+
+- Spawn N workers to stress test CPU:
```

Comment:
Let us use concrete numbers instead of generic ones for N. Please use 4,2,1 interchangeably throughout the examples.